openpype/pype_commands.py

```diff
@@ -186,6 +186,7 @@ def extractenvironments(output_json_path, project, asset, task, app,
                 app,
                 env_group=env_group,
                 launch_type=LaunchTypes.farm_render,
+                env={}
```

Comment:
I've noticed that behaviour before was that `extractenvironments` added all environment - basically whole `os.environ` and that wasn't intended. This is caused by this https://github.com/ynput/OpenPype/blob/ddefee8f0e05320ea9c3d8a5d63d196517db747b/openpype/lib/applications.py#L987-L997

```python
        if src_env is None:
            src_env = os.environ
```

but `extractenvironments` should work only with stuff added by OpenPype. So far it worked because `GlobalJobPreload` that is calling this is always executed on the same machine that is also rendering. But if you want to use this same functionality elsewhere, it won't. So this change was something with (hopefully) minimal side-effects.